src/__tests__/Hooks-test.js

```diff
@@ -19,118 +33,169 @@ describe('Hooks', () => {
+    expect(RESTController.ajax.mock.calls[0][1]).toBe('https://api.parse.com/1/hooks/functions');
   });
 
-  it('shoud properly build GET triggers', () => {
+  it('should properly build GET triggers', async () => {
     Hooks.getTriggers();
 
     expect(CoreManager.getHooksController().sendRequest.mock.calls[0]).toEqual([
       'GET',
-      '/hooks/triggers',
+      'hooks/triggers',
     ]);
+    defaultController.sendRequest = sendRequest;
+    await Hooks.getTriggers();
+    expect(RESTController.ajax.mock.calls[0][1]).toBe('https://api.parse.com/1/hooks/triggers');
   });
 
-  it('shoud properly build GET function', () => {
+  it('should properly build GET function', async () => {
     Hooks.getFunction('functionName');
 
     expect(CoreManager.getHooksController().sendRequest.mock.calls[0]).toEqual([
       'GET',
-      '/hooks/functions/functionName',
+      'hooks/functions/functionName',
     ]);
+    defaultController.sendRequest = sendRequest;
+    await Hooks.getFunction('functionName');
+    expect(RESTController.ajax.mock.calls[0][1]).toBe(
+      'https://api.parse.com/1/hooks/functions/functionName'
+    );
   });
 
-  it('shoud properly build GET trigger', () => {
+  it('should properly build GET trigger', async () => {
     Hooks.getTrigger('MyClass', 'beforeSave');
 
     expect(CoreManager.getHooksController().sendRequest.mock.calls[0]).toEqual([
       'GET',
-      '/hooks/triggers/MyClass/beforeSave',
+      'hooks/triggers/MyClass/beforeSave',
     ]);
+    defaultController.sendRequest = sendRequest;
+    await Hooks.getTrigger('MyClass', 'beforeSave');
+    expect(RESTController.ajax.mock.calls[0][1]).toBe(
+      'https://api.parse.com/1/hooks/triggers/MyClass/beforeSave'
+    );
   });
 
-  it('shoud properly build POST function', () => {
+  it('should properly build POST function', async () => {
     Hooks.createFunction('myFunction', 'https://dummy.com');
 
     expect(CoreManager.getHooksController().sendRequest.mock.calls[0]).toEqual([
       'POST',
-      '/hooks/functions',
+      'hooks/functions',
       {
         functionName: 'myFunction',
         url: 'https://dummy.com',
       },
     ]);
+    defaultController.sendRequest = sendRequest;
+    await Hooks.createFunction('myFunction', 'https://dummy.com');
+    expect(RESTController.ajax.mock.calls[0][1]).toBe('https://api.parse.com/1/hooks/functions');
   });
 
-  it('shoud properly build POST trigger', () => {
+  it('should properly build POST trigger', async () => {
     Hooks.createTrigger('MyClass', 'beforeSave', 'https://dummy.com');
 
     expect(CoreManager.getHooksController().sendRequest.mock.calls[0]).toEqual([
       'POST',
-      '/hooks/triggers',
+      'hooks/triggers',
       {
         className: 'MyClass',
         triggerName: 'beforeSave',
         url: 'https://dummy.com',
       },
     ]);
+    defaultController.sendRequest = sendRequest;
+    await Hooks.createTrigger('MyClass', 'beforeSave', 'https://dummy.com');
```

Comment:
```suggestion
    await Hooks.createTrigger('MyClass', 'beforeSave', 'https://example.com');
```

We are using `example.com` for all example domains in tests and docs.

Reply:
Done